### DISCUSSION TITLE
Revert changes added due to bug in ImportC

### DISCUSSION THREAD

**Discussion by aremmell** [aremmell/libsir#471] [Ideas]
Revert ImportC-related changes from PR #390

**Comment by johnsonjh** [comment]
Tested and still an issue as of v2.108.0-beta1.  I'll revisit after the next release.

**Comment by johnsonjh** [comment]
Looks OK now with 2.108.0-rc1, so I'll revisit this after 2.108.0 is released and available through the regular channels for all platforms.

**Comment by aremmell** [comment]
How bout now?

**Comment by johnsonjh** [comment]
I guess we’d have to raise the base supported version to at least 2.108.0 or something.  OpenBSD currently has ported 2.109.0 now.

I see 2.110.0 is out now.  I’ll check that it works right on all the platforms today or tomorrow.  I’ll set a reminder for myself.

**Comment by johnsonjh** [comment]
@aremmell I tried it with #451 but while 64-bit worked, 32-bit still failed:

![Image](https://github.com/user-attachments/assets/daf4f6e0-b301-4671-b2d5-7460ee7ab3be)

I’ll close that PR for now and see if I can re-open the issue upstream.
